b2b_ecommerce/views.py

```diff
@@ -177,20 +178,28 @@ def get(self, request, *args, **kwargs):
 
         rows = (
             {
-                "code": code,
                 "url": make_checkout_url(
                     code=code,
                     product_id=order.product_version.text_id,
                     run_tag=order.program_run.run_tag if order.program_run else None,
-                ),
+                )
             }
             for code in Coupon.objects.filter(
                 versions__payment_version__b2border=order
             ).values_list("coupon_code", flat=True)
         )
 
-        return make_csv_http_response(
-            csv_rows=rows, filename=f"enrollmentcodes-{order_hash}.csv"
+        instructions = [
+            "Distribute the links below to each of your learners. Additional instructions are available at:",
+            '=HYPERLINK("https://xpro.zendesk.com/hc/en-us/articles/360048166292-How-to-I-distribute-my-enrollment-codes-that-I-purchased-in-a-bulk-order-")',
+        ]
+        instuctions_generator = (
+            {"": instructions[i]} for i in range(len(instructions))
+        )
+
+        return make_csv_http_response_with_instructions(
+            csv_rows=chain(instuctions_generator, rows),
+            filename=f"enrollmentcodes-{order_hash}.csv",
```

Comment:
Please update the method to take `instructions` in a separate parameter as requested later on in this review.